docs/src/integrations/exchange.md

```diff
@@ -177,9 +177,9 @@ transfer to the appropriate deposit address.
 
 ### Poll for Blocks
 
-The easiest way to track all the deposit accounts for your exchange is to poll
-for each confirmed block and inspect for addresses of interest, using the
-JSON-RPC service of your Solana api node.
+To track all the deposit accounts for your exchange, poll for each confirmed
+block and inspect for addresses of interest, using the JSON-RPC service of your
+Solana api node.
```

Comment:
```suggestion
Solana API node.
```

May as well pick nits while we're here :wink:

Reply:
For sure! Lay 'em on me